ios/Plugin/CapacitorUrlRequest.swift

```diff
@@ -88,24 +96,26 @@ public class CapacitorUrlRequest: NSObject, URLSessionTaskDelegate {
     }
     
     func getRequestData(_ body: JSValue, _ contentType: String) throws -> Data? {
-        if contentType.contains("application/json") {
+        // If data can be parsed directly as a string, return that without processing.
+        if let strVal = try? getRequestDataAsString(body) {
```

Comment:
Not sure if this is correct, or better said, what `getRequestDataAsString` does. JSON is for me also a valid string value and would be treated as string here.

Reply:
skip